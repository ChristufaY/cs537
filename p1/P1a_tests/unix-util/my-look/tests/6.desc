Case non-sensitivity test: partially capitalized input and output
