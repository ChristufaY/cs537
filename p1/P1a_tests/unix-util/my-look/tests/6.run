./my-look -f ./tests/wordlist.txt DeSICc
