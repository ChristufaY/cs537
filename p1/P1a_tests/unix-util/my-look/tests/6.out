desiccant
desiccate
desiccated
desiccation
desiccative
